linux/scripts/upload-to-debian.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Upload the latest release to mentors.debian.net. Call from stable-* branch.
+# Usage: upload-to-debian.sh -k <Debian signing key>
+# Requires an entry in ~/.dput.cf:
+# ```
+# [mentors]
+# fqdn = mentors.debian.net
+# incoming = /upload
+# method = https
+# allow_unsigned_uploads = 0
+# progress_indicator = 2
+# allowed_distributions = .*
+# ```
+
+set -e
```

Comment:
```suggestion
set -eu
```

For consistency in our bash scripts, and also because it helps trap typos. But does make working with unset variables much more tedious -- and you'd have to fix up all of your refs to your param vars `NOOP`, `PUSH`, `debkeyid`. So consider this optional...

I note also that `fail()` in shellHelperFunctions.sh -- see later comments -- can barf with `-u` so one day should fix that too...